src/WebJobs.Script.Grpc/Server/GrpcServer.cs

```diff
@@ -46,7 +57,7 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                    _server.ShutdownAsync();
+                    ShutdownAsync();
```

Comment:
Was this meant to be fire-and-forget? We've never been awaiting it...

Reply:
yes., best case effort